scripts/upload-source-maps.js

```diff
@@ -1,65 +1,88 @@
 'use strict'
 
+const path = require('path')
 const { getSecretKey, command, printLog, runMain } = require('./utils')
+const { buildRootUploadPath, buildDatacenterUploadPath, buildSourcePath, packages } = require('./deployment-utils')
 const { SDK_VERSION } = require('./build-env')
 
 /**
  * Upload source maps to datadog
  * Usage:
- * BUILD_MODE=canary|release node upload-source-maps.js staging|canary|vXXX
+ * BUILD_MODE=canary|release node upload-source-maps.js staging|canary|vXXX root,us1,eu1,...
```

Comment:
IMO, the two upload strategies that we have (root or datacenter) need to be clearly identified in the scripts since it define their behaviours.
Making the root upload the default one seems odd to me since eventually we will only have it for `staging` and `canary`.

Reply:
Ok!